events/guildMemberAdd.js

```diff
@@ -0,0 +1,22 @@
+const players = require("../schemas/players")
+const { ids } = require("../config")
+
+module.exports = (client) => {
+    client.on("guildMemberAdd", async (member) => {
+        let sim = client.servers.cache.get(ids.server.sim)
+        if (member.guild.id !== sim.id) return
+        client.invites.every(async (invite) => {
+            let inv = await sim.invites.resolve(invite.code)
+            if(inv.uses !== invite.uses) {
+                await players.updateOne({"badges.invite.code": invite.code}, {$inc:{"badges.invite.members": 1}})
+                let guy = await players.findOne({"badges.invite.code": invite.code})
+                if(guy.badges.invite.unlocked === true) return
+                if(guy.badges.invite.members = 15) {
+                    await players.updateOne({"badges.invite.code": invite.code}, {$set:{"badges.invite.unlocked": true}})
+                    let guyUser = client.users.resolve(guy.user)
+                    guyUser.send({content:`Congratulations! You just unlocked the invite badge with the code \`${invite.code}\``})
```

Comment:
Use i10n for this

Reply:
So I should replace every string with an i10n string? or just this one?

And this also implements creating prs on i10n right?

Reply:
Just the strings that are public, not the error stuff for staff.
PR it to the bottom of the en.json file for the English translation